opencti-platform/opencti-front/src/private/components/workspaces/Workspaces.tsx

```diff
@@ -102,6 +106,10 @@ const Workspaces: FunctionComponent<WorkspacesProps> = ({
   const { isFeatureEnable } = useHelper();
   const FAB_REPLACED = isFeatureEnable('FAB_REPLACEMENT');
 
+  const theme = useTheme();
+  const { settings } = useContext(UserContext);
+  const importFromHubUrl = isNotEmptyField(settings) ? `${settings.platform_xtmhub_url}/redirect/custom_dashboard?octi_instance_id=${settings.id}`.replaceAll('//', '/') : '';
```

Comment:
We should decide what to do if the url si not present. Display the button with no href ? Hide the button ? I can't recall what we have decided regarded airgapped platform (that could have a conf with no xtmhub url)

Reply:
I added this because `settings` can be `undefined` during the loading time.